src/syncStripeWithAirtable.ts

```diff
@@ -0,0 +1,99 @@
+import { TriggerClient } from "@trigger.dev/sdk";
+import { Stripe } from "@trigger.dev/stripe";
+import { Airtable } from "@trigger.dev/airtable";
+
+const client = new TriggerClient({ id: "jobs-showcase" });
+
+const stripe = new Stripe({
+  id: "stripe",
+  apiKey: process.env.STRIPE_API_KEY!,
+});
+
+const airtable = new Airtable({
+  id: "airtable",
+  token: process.env.AIRTABLE_TOKEN!,
+});
+
+type Customers = {
+  stripe_customer_id?: string;
+  Sales?: string[];
+};
+
+type Sales = {
+  payment_intent_id: string;
+  amount: number;
+  currency: string;
+  Customers?: string[];
+  "stripe_customer_id (from Customers)": string;
+};
+
+client.defineJob({
+  id: "stripe-new-sale-update-airtable",
+  name: "On new Stripe sale update Airtable",
+  version: "1.0.0",
+  integrations: {
+    stripe,
+    airtable,
+  },
+  trigger: stripe.onPaymentIntentSucceeded(),
+  run: async (payload, io, ctx) => {
+    const customersTable = io.airtable
```

Comment:
I wonder if we could create some kind of Airtable base template that can be easily cloned and used for this?